nemo/core/connectors/save_restore_connector.py

```diff
@@ -424,16 +425,28 @@ def _handle_artifacts(self, model, nemo_file_folder):
 
         # Process current tarfile artifacts by unpacking the previous tarfile and extract the artifacts
         # that are currently required.
+        restoration_paths = []  # model + submodules restoration paths
```

Comment:
These changes are separated to https://github.com/NVIDIA/NeMo/pull/5671 (nested NeMo models)